project-base/docker/nginx/nginx-google-cloud.conf

```diff
@@ -0,0 +1,66 @@
+upstream php-upstream {
+    server php-fpm:9000;
+}
+
+server {
+    listen 8080;
+    access_log /var/log/nginx/shopsys-framework.access.log;
+    root /var/www/html/web;
+    server_tokens off;
+
+    location ~ /\. {
+        # hide dotfiles (send to @app)
+        try_files @app @app;
+    }
+    location ~ ^/index.php$ {
+        # hide index.php (send to @app)
+        try_files @app @app;
+    }
+    location / {
+        # try to serve existing files directly, fallback to @app
+        try_files $uri @app;
+
+        # Enable compression of JavaScripts and CSS
+        location ~ (\.js|\.css)$ {
+            gzip on;
+            gzip_types application/javascript text/css;
+
+            # Cache versioned static content for 1 year
+            location ~ ((-v[0-9]+\.js)|(_[0-9]+\.css))$ {
+                expires 1y;
+
+                # (try_files directive is not inherited from the parent location so we have to repeat it)
+                try_files $uri @app;
+            }
+
+            # (try_files directive is not inherited from the parent location so we have to repeat it)
+            try_files $uri @app;
+        }
+
+        location ~ ^/content(-test)?/images/ {
```

Comment:
not sure whether it makes sense

Reply:
i would leave it to another US or issue?